### PR TITLE
Changed subject to object and added an ASCII pic

### PR DESCRIPTION
**Pull request by floridop**
Changed content of your story. You are free to ask me to do any change you like before approving!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,54 @@
 # MNXB01-learn
 
-Once upon a time, there was a **beautiful** _princess_
+Once upon a time, there was a **nightime** _birthday cake_
```
Comment (jesgum) [inline README.md]: Change **nightime** to **nighttime**

**Review by jesgum — CHANGES_REQUESTED** [review]
Please add a description of the favor of the nighttime cake

**Comment by floridop** [comment]
Thanks for the correction! In your message, I guess you meant _flavor_. We are apparently missing letters in this process :)

File: README.md (hunk 1 above)
Comment (floridop) [reply README.md]: Ok thanks for the correction!

**Comment by jesgum** [comment]
Ah yes! It HAS to be a bug. I mean there is NO WAY both of us missed letters :-]

**Comment by floridop** [comment]
Changes implemented in commit [2e359f](https://github.com/jesgum/MNXB01-learn/pull/2/commits/2e359fca4b69bb65c0dd84d3b25fb7de8765a3f3)

**Comment by floridop** [comment]
I'll close this part of the homework, but if there is something else 
you're curious to try and I am available I'll be happy to test.

**Comment by jesgum** [comment]
I think I'm happy with these interactions we've done.

**Pull request merged**